app/views/donation_bands/show.slim

```diff
@@ -0,0 +1,8 @@
+h3 = @donation_band.name
+
+= link_to t('donation_bands.show.edit'), edit_donation_band_path(@donation_band), class: 'btn btn-sm btn-warning'
+= link_to t('donation_bands.show.destroy'), destroy_donation_band_path(@donation_band), class: 'btn btn-sm btn-danger', data: {warning: 'Are you sure?'}
+
+div
+  | Donation values in this band:
+  = @donation_band.amounts
```

Comment:
This view is raising a `NoMethodError` on `destroy_donation_band_path`. Do you think this view can be deleted, since there's no link to it anyway?